packages/xo-server/src/api/network.mjs

```diff
@@ -120,3 +120,19 @@ delete_.params = {
 delete_.resolve = {
   network: ['id', 'network', 'administrate'],
 }
+
+// =================================================================
+
+export async function updatePurpose({ networkId, oldPurpose, newPurpose }) {
```

Comment:
Instead of exposing `purpose`, expose `nbd` and `insecureNbd` to mirror what has been done in `xapi-objects-to-xo.mjs`.